plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseSharedMetastoreTest.java

```diff
@@ -128,4 +134,62 @@ public void testShowSchemas()
                 showCreateIcebergWithRedirectionsSchema,
                 getExpectedIcebergCreateSchema("iceberg_with_redirections"));
     }
+
+    @Test
+    public void testTimeTravelWithRedirection()
+            throws InterruptedException
+    {
+        try {
+            assertUpdate(format("CREATE TABLE iceberg.%s.nation_test AS SELECT * FROM nation", schema), 25);
```

Comment:
I think this table affects other tests using `information_schema`. `testReadInformationSchema` & `testShowTables` will fail if they are executed at the same time.